src/main/java/com/minecolonies/coremod/entity/pathfinding/MinecoloniesAdvancedPathNavigate.java

```diff
@@ -326,7 +338,7 @@ public void setSpeed(final double d)
             Log.getLogger().error("Tried to set a too high speed for entity:" + ourEntity, new Exception());
             return;
         }
-        walkSpeed = d;
+        speed = d;
```

Comment:
this will be ignored as speed is the temporary variable which frequently gets updated

Reply:
Yeah the general structure should be maintained.

Reply:
Whoops, I hade some different code there before but forgot to change it back, will fix it again